File: PHP Excel Examples/CellsApiDemo/Cell/setCellValue.php
<?php

use Spire\Cloud\Excel\Sdk\Api\CellsApi;
use Spire\Cloud\Excel\Sdk\Configuration;

$appId = "your id";
$appKey = "your key";
$baseUrl = "https://api.e-iceblue.cn";
$configuration = new Configuration($appId, $appKey, $baseUrl);
$apiInstance = new CellsApi($configuration);

$name = "SetCellValue.xlsx";
$folder = "input";
$storage = null;
$sheetName = "Sheet1";
$cellName = "C20";
$value = "1";
$type = "String";
$formula = null;
$apiInstance->setCellValue($name, $sheetName, $cellName, $value, $type, $formula, $folder, $storage);

File: Python Excel Examples/CellsApiDemo/cell/setCellValue.py
import spirecloudexcel
from spirecloudexcel.configuration import Configuration as ExcelConfiguration
from spirecloudexcel.api.cells_api import CellsApi

appId = "your id"
appKey = "your key"
baseUrl = "https://api.e-iceblue.cn"
configuration = ExcelConfiguration(appId, appKey,baseUrl)
api = spirecloudexcel.api.cells_api.CellsApi(configuration)

name = "SetCellValue_1.xlsx"
folder = "/Cells/Cell/Value/"
storage = ""
sheetName = "Sheet1"
cellName = "C20"
value = "1"
type = "string"
formula = ""
api.set_cell_value(name, sheetName, cellName, value, type=type, formula=formula, folder=folder, storage=storage)